pytorch_lightning/core/lightning.py

```diff
@@ -1619,7 +1619,7 @@ def get_progress_bar_dict(self):
         module_tbptt_enabled = self.truncated_bptt_steps > 0
         trainer_tbptt_enabled = self.trainer.truncated_bptt_steps is not None and self.trainer.truncated_bptt_steps > 0
         if module_tbptt_enabled or trainer_tbptt_enabled:
-            tqdm_dict["split_idx"] = self.trainer.split_idx
+            tqdm_dict["split_idx"] = self.trainer.train_loop.split_idx
```

Comment:
is `get_progress_bar_dict` still needed on the lightning module? wouldn't things come from `self.log` ?

Reply:
it's for users to override and customize the default elements in the progress bar. Like how the version number is displayed or apparently the split index  here.

Reply:
It's not something one could customize through self.log directly I would say.

Reply:
Is it customization we would want to push to a custom progress bar callback instead of being part of the core module?

Reply:
> custom progress bar callback instead of being part of the core module?
IMO yes for maximum separation of concerns

But people might complain about having to define a custom progress bar just for this

Reply:
one could argue maybe the split_idx is not very useful to display in the progbar but still I would keep the hook